src/components/Books.jsx

```diff
@@ -0,0 +1,75 @@
+import { useSelector, useDispatch } from 'react-redux';
+import React, { useState } from 'react';
+import BookList from './BookList';
+import { addBook } from '../redux/slices/books/bookSlice';
+
+function Books() {
+  const { books } = useSelector((store) => store.book);
+  const dispatch = useDispatch();
+  const [author, setAuthor] = useState('');
+  const [title, setTitle] = useState('');
+  const [cat, setCat] = useState('Fiction');
+
+  return (
+    <div>
+      <BookList books={books} />
+      <div style={{
+        width: '100%',
+        background: 'black',
+        height: '2px',
+        marginTop: '10px',
+        marginBottom: '10px',
+      }}
+      />
+      <form>
+        <h1>ADD NEW BOOK</h1>
+        <div>
+          <input
+            type="text"
+            value={author}
+            placeholder="author"
+            onChange={(e) => setAuthor(e.target.value)}
+            required
+          />
+          <input
+            type="text"
+            value={title}
+            placeholder="title"
+            onChange={(e) => setTitle(e.target.value)}
+            required
+          />
+          <select
+            value={author}
+            onChange={(e) => setCat(e.target.value)}
+          >
+            <option value="action">Fiction</option>
+            <option value="sci-fi">Nonfiction</option>
+            <option value="Economy">Drama</option>
+          </select>
```

Comment:
- [ ] To make your codebase clean and your components reusable, I will recommend that you create a separate component called <NewBookForm /> which will contain the form to add a book.  This way, your code will be modular and readable.